From b24afabe9dcf3ecaba66ca98f5929b3b94535617 Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jslaby@novell.com>
Date: Wed, 11 Nov 2009 17:28:06 +0100
Subject: [PATCH 25/25] x86: add ia32 compat prlimit syscalls
References: FATE#305733

To support 32/64-bit compatibility (rlimit structure contains 2 longs)
for prlimit syscalls, add compat wrappers for them.

Signed-off-by: Jiri Slaby <jslaby@novell.com>
Cc: James Morris <jmorris@namei.org>
Cc: Heiko Carstens <heiko.carstens@de.ibm.com>
Cc: Andrew Morton <akpm@linux-foundation.org>
Cc: Ingo Molnar <mingo@elte.hu>
Automatically created from "patches.suse/rlim-0025-x86-add-ia32-compat-prlimit-syscalls.patch" by xen-port-patches.py

--- head-2009-12-04.orig/arch/x86/ia32/ia32entry-xen.S	2009-11-06 14:53:39.000000000 +0100
+++ head-2009-12-04/arch/x86/ia32/ia32entry-xen.S	2009-12-04 12:04:06.000000000 +0100
@@ -734,4 +734,6 @@ ia32_sys_call_table:
 	.quad compat_sys_pwritev
 	.quad compat_sys_rt_tgsigqueueinfo	/* 335 */
 	.quad sys_perf_event_open
+	.quad compat_sys_getprlimit
+	.quad compat_sys_setprlimit
 ia32_syscall_end:
